Sources/SwiftGraphQLCodegen/Generator/Field.swift

```diff
@@ -91,7 +91,9 @@ extension Field {
         if isDeprecated {
             // NOTE: It's possible that a string contains double-quoted characters in deprecation reason.
             //       http://spec.graphql.org/October2021/#sec-Language.Directives
-            let message = deprecationReason?.replacingOccurrences(of: "\"", with: "\\\"") ?? ""
+            let message = deprecationReason?
+                .trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
The Swift version also can support multiple line messages, which can improve readability so I'd suggest we retain that?

```swift
@available(*, deprecated, message: """
Line 1
Line 2
""")
```

Reply:
Yes, let's do that!